docs/guides/configuration.rst

```diff
@@ -19,6 +19,10 @@ Here is a list of all available options
 +-----------------+----------------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
 | dbUrl           | –                                | ``--dbUrl``              | ``hoodie_dbUrl``         | If provided, uses external CouchDB. URL has to contain credentials. Sets ``dbAdapter`` to ``pouchdb-adapter-http``  |
 +-----------------+----------------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
+| dbUrlUsername   | –                                | ``dbUrlUsername``        | ``hoodie_dbUrlUsername`` | If ``dbUrl`` is set, you can use ``dbUrlUsername`` to set the username to use when making requests to CouchDB       |
++-----------------+----------------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
+| dbUrlPassword   | –                                | ``dbUrlPassword``        | ``hoodie_dbUrlPassword`` | If ``dbUrl`` is set, you can use ``dbUrlPassword`` to set the password to use when making requests to CouchDB       |
++-----------------+----------------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
```

Comment:
Great catch! Can you adopt the description for `dbUrl`? So instead of "URL has to contain credentials" we could say sth like "URL has to contain credentials, or `dbUrlUsername` and `dbUrlPassword` have to be set separately"

Reply:
Makes sense. I'll modify the language and push the change.